Content.Server/Atmos/Piping/Unary/EntitySystems/GasVentPumpSystem.cs

```diff
@@ -65,43 +64,71 @@ private void OnGasVentPumpUpdated(EntityUid uid, GasVentPumpComponent vent, Atmo
                 return;
             }
 
-            if (vent.PumpDirection == VentPumpDirection.Releasing)
+            var timeDelta =  (_gameTiming.CurTime - device.LastProcess).TotalSeconds;
```

Comment:
How does LastProcess work if the device is turned off then turned back on 10 minutes later?

Reply:
`AtmosDeviceSystem` currently just keeps updating LastProcess for as long as the device is part of some atmosphere, and should set it when joining one. ATM disabled devices just ignore the `AtmosDeviceUpdateEvent`s that the system raises.

So AFAIK it should be fine, though `AtmosDeviceSystem` doesn't currently seems to do anything on pause/unpause and just ignores that? So that probably needs fixing separately.